app/src/main/java/com/trunghoang/aroundhere/ui/adapter/PlacesAdapter.java

```diff
@@ -0,0 +1,77 @@
+package com.trunghoang.aroundhere.ui.adapter;
+
+import android.content.res.Resources;
+import android.support.annotation.NonNull;
+import android.support.v7.widget.RecyclerView;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.ImageView;
+import android.widget.TextView;
+
+import com.squareup.picasso.Picasso;
+import com.trunghoang.aroundhere.R;
+import com.trunghoang.aroundhere.data.model.Place;
+
+import java.util.List;
+
+public class PlacesAdapter extends RecyclerView.Adapter<PlacesAdapter.PlaceViewHolder> {
+    private List<Place> mPlaces;
+
+    public PlacesAdapter(List<Place> places) {
+        mPlaces = places;
+    }
+
+    @NonNull
+    @Override
+    public PlaceViewHolder onCreateViewHolder(@NonNull ViewGroup parent, int i) {
+        View v = LayoutInflater.from(parent.getContext()).inflate(R.layout.recycler_item, parent, false);
+        return new PlaceViewHolder(v);
+    }
+
+    @Override
+    public void onBindViewHolder(@NonNull PlaceViewHolder placeViewHolder, int position) {
+        placeViewHolder.bindView(mPlaces.get(position));
+    }
+
+    @Override
+    public int getItemCount() {
+        return mPlaces == null ? 0 : mPlaces.size();
+    }
+
+    static class PlaceViewHolder extends RecyclerView.ViewHolder {
+        private ImageView mPlacePhoto;
+        private TextView mDistance;
+        private TextView mTextOpen;
+        private TextView mTitle;
+        private TextView mAddress;
+
+        public PlaceViewHolder(@NonNull View itemView) {
+            super(itemView);
+            mPlacePhoto = itemView.findViewById(R.id.image_place_photo);
+            mDistance = itemView.findViewById(R.id.text_distance);
+            mTextOpen = itemView.findViewById(R.id.text_is_open);
+            mTitle = itemView.findViewById(R.id.text_title);
+            mAddress = itemView.findViewById(R.id.text_address);
+        }
+
+        public void bindView(Place place) {
+            if (place != null) {
```

Comment:
đoạn code này e có thể viết 
```suggestion
            if (place == null) {
            return;
            }
```